### PR TITLE
Simpler CLI for starting the webcm

### PR DESCRIPTION
**Pull request by kuba-orlik**
- start a static website server if the user doesn't provide their own proxy target
- do not require a config file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,10 +22,11 @@ It's very easy to get up and running with WebCM using `npx`!
 
 You might want to make WebCM load a locally developed Managed Component.
 
-1. Place your Managed Component inside a `components` directory next to your config file
```
Comment (bjesus) [inline README.md:25]: should we keep the information about the config-file based way of loading WebCM in the README somewhere? It still has its use cases I think (e.g. I want to load two components, I want to try out permissions).

File: assets/index.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>My website</title>
+  </head>
+  <body>
+    <h1>Hello, world!</h1>
+    <p>
+      Eos assumenda ut qui non. Ut et et aut repellendus ut saepe maxime
```
Comment (bjesus) [inline assets/index.html:9]: Let's make this a page with some more actual information. Eg. "This page is running WebCM, do X to have WebCM proxy your own site, use `webcm.X` to send events" etc.

File: bin/webcm.js [hunk 3]
```diff
@@ -96,8 +103,13 @@ function onFatalError(error) {
           transpileOnly: true,
           dir: __dirname,
         })
-        const { startServer } = require('../lib/server')
-        startServer(argv.config, argv.components)
+        const { startServerFromConfig } = require('../lib/server')
+        startServerFromConfig({
```
Comment (bjesus) [inline bin/webcm.js:107]: did we do this thing that allows every config setting to be set using a CLI argument, or did we give up on it for now?

**Comment by kuba-orlik** [comment]
I've discovered some more things to take care of, gonna update this PR soom

File: README.md [hunk 4]
```diff
@@ -1,33 +1,75 @@
...
+(see [example.config.ts](./example.config.ts)) and set it to:
+
+```ts
+export default {
+  components: [
+    {
+      path: './path/to/component.ts',
+      permissions: ['execute_unsafe_scripts'],
+    },
+  ],
```
[the remark this answers is not in the document]
Comment (kuba-orlik) [reply README.md:65]: I've confirmed that auto-downloading is left intact

File: assets/index.html [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  <body>
+    <h1>Hello, WebCM!</h1>
+    <p>This page is running WebCM.</p>
+
+    <p>To test Managed Components with WebCM on a different website, run:</p>
+
+    <pre><code>$ npx webcm start -u https://example.com</code></pre>
+
+    <p>
+      You can run <code>webcm.track({ example: 'data' })</code> to manually
```
[the remark this answers is not in the document]
Comment (kuba-orlik) [reply assets/index.html]: Fixed!

**Review by bjesus — APPROVED** [review]

**Comment by kuba-orlik** [comment]
There's currently a problem that makes webcm not polyfill `crypto` properly, fixing that...

**Review by jonnyparris — APPROVED** [review]

**Pull request merged**